lib/api/src/modules/notifications.ts

```diff
@@ -3,7 +3,15 @@ import { ModuleFn } from '../index';
 export interface Notification {
   id: string;
   link: string;
-  content: string;
+  content: {
+    headline: string;
+    subHeadline?: string;
```

Comment:
Should we allow ReactNode here? So `span`, `strong` etc is possible?

Reply:
Good point. I can't see a good reason to limit this to just string. Will add ReactNode too subHeadline and add an example for it.

Reply:
Quick update: added `ReactNode` to `subHeadline` but kept `headline` as `string` to be able to have a title attribute for the headline: https://github.com/storybookjs/storybook/pull/12320/files#diff-f0193139de5892a0c79f1452e9f8bc49391f4aab7f7d947d83f05d53a13e9426R92

Reply:
👍